### ISSUE TITLE
Settings not saving through update

### ISSUE THREAD

**Issue by FlyingGardenGnomeStudios** [thomst08/requestrr#32]
For my TrueNAS installation, for the last 2 years each time I update the app, all the settings for connecting to discord and the series/movie categories are reset.

**Comment by thomst08** [comment]
Hey @FlyingGardenGnomeStudios,

It sounds like you have the config folder incorrect setup in your TrueNAS settings.
You will need to make sure there is a volume setup for ``/root/config`` and is pointing to a local folder in TrueNAS.

**Comment by thomst08** [comment]
Hey @FlyingGardenGnomeStudios,

I haven't heard from you regarding this issue, did you have a chance to check your volume settings?

**Comment by FlyingGardenGnomeStudios** [comment]
OK, I found the issue.
The first set up states that any setting besides PVC may break the app.
From your suggestion, I configured it to a drive I use for some other app settings which is an SSD different from the app installation drive but that caused the app to fail loading.
Ultimately I moved it to the same drive the app is installed in it's own config dataset and this appears to be taking the update and maintaining settings.

Hopefully any future noob will find this helpful.
